config/routes.rb

```diff
@@ -47,8 +47,7 @@
     get 'users/auth/webauth' => 'sessions#new', :as => 'webauth_login'
     get 'users/auth/webauth/logout' => 'sessions#destroy_webauth', :as => 'webauth_logout'
   end
-  get 'error' => 'signin#error', :as => 'error'
-  get 'contact' => 'application#contact', :as => 'contact'
+  match 'contact' => 'application#contact', :as => 'contact', via: [:get, :post]
```

Comment:
No need to fix, but I wish this was going to ContactController instead.

Reply:
created a new contact controller and moved tests too